hokusai/staging.yml

```diff
@@ -27,58 +27,84 @@ spec:
         layer: application
       name: {{ project_name }}-api
     spec:
+      initContainers:
+      - name: setenv
+        image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/fortress:staging
```

Comment:
I left a similar comment on motion but I don't entirely understand why we're using fortress staging for staging deployments, could that not break staging deployments if something goes wrong with fortress?

Reply:
> I don't entirely understand why we're using fortress staging for staging deployments

as opposed to using Fortress prod?

yeah, i may be setting a precedent here. the idea is:
- projects' staging.yml point to Fortress staging Docker image
- projects' production.yml point to Fortress  prod Docker image
- merging Fortress PR updates Fortress staging Docker image, affects projects' Staging deployments immediately. we wait a few days for problems to surface in Staging, before releasing Fortress Deploy PR. (Horizon automatic release is disabled for Fortress)